taui/src/components/draw-neighborhood-bounds.js

```diff
@@ -18,7 +17,12 @@ export default class DrawNeighborhoodBounds extends React.PureComponent {
   getTooltip = (feature) => {
     if (!feature || !feature.properties) return ''
     const {routable, town} = feature.properties
-    return routable ? town : town + ' ' + message('Map.Unreachable')
+    const tooltipEl = document.createElement('div')
```

Comment:
Why not use `React.createElement()` or similar react/JSX templating?

Reply:
I tried using JSX at first but it didn't work. I believe this is because the code in `getTooltip()` isn't called as part of the React rendering cycle. Its return value needs to be either a [string or an HTMLElement](https://leafletjs.com/reference-1.0.0.html#layer-bindtooltip), whereas `React.createElement()` would return a React element.